src/xbps/repositories/index.md

```diff
@@ -8,7 +8,20 @@ which may also be signed.
 Note that, while local repositories do not require signatures, remote
 repositories *must* be signed.
 
-## Official Repositories
+## The main repository
+
+The locations of the main repository in relation to a base [mirror
+URL](./mirrors/index.md) are:
+
+| System type | Location           |
+|-------------|--------------------|
+| glibc       | `/current`         |
+| musl        | `/current/musl`    |
+| aarch64     | `/current/aarch64` |
```

Comment:
Not sure these should go in the same table... Or perhaps have `aarch64` and `aarch64-musl`  in the last line?

Reply:
Fair point. So two tables, one of which is for `aarch64` specifically, and which lists both `/current/aarch64` and `/current/aarch64/musl`?

Reply:
I don't think so, because i's `/current/aarch64` for both. Perhaps `aarch64 and aarch64-musl | /current/aarch64` ?

Reply:
i've changed the text as you say, and presented the information as a list instead of a table - WDYT?

Reply:
Looks good to me, thanks!